salt/utils/schedule.py

```diff
@@ -210,7 +210,7 @@ def _check_max_running(self, func, data, opts, now):
         # dict we treat it like it was there and is True
 
         # Check if we're able to run
-        if not data['run']:
+        if not data.get('run', True):
```

Comment:
Why is it that we want to default to `True` here?

Reply:
Updated the code, reverting this one back and setting the value in the run function.